polyfill/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "temporal",
+  "version": "1.0.0",
+  "description": "TC39 Temporal Proposal Polyfill",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "for FILE in $(find test -type f -name '*.js' | grep -v 'test/node.js'); do node -r ./test/node.js $FILE; done",
+    "build": "rollup -c rollup.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://bbgithub.dev.bloomberg.com/javascript-guild/temporal-polyfill.git"
+  },
+  "keywords": [
+    "TC39",
+    "Polyfill",
+    "Temporal"
+  ],
+  "author": "Philipp Dunkel <pdunkel1@bloomberg.net>",
+  "license": "UNLICENSED",
+  "dependencies": {},
+  "devDependencies": {
+    "rollup": "^0.59.1",
```

Comment:
do we need to use rollup here? that prevents people from deep-importing just the parts they need.